### ISSUE TITLE
Color contrast on Preassembly red buttons has changed, making text nearly unreadable

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/pre-assembly#1424]
**Describe the bug**
The color contrast and hover behavior on the Preassembly buttons has changed, rendering the text nearly illegible and non-accessible. The text is now dark with a red or black background and then the background disappears on hover. 

https://github.com/sul-dlss/pre-assembly/assets/8975583/067d707b-4463-4f23-a89a-1ec2e4f6a8ab

**To Reproduce**
See the screen recording above. I think it covers most of the buttons in the interface.

**Expected behavior**
The text behind the red buttons should be white. I'm not sure why the "New Discovery Report" (visible at the end of the screen recording) is now a black button. I can't remember if it looked different than the "Run Preassembly" button right next to it. Hopefully it will be possible to determine what the button colors and behavior used to be.

**Additional context**
I don't think we've made a any changes to Preassembly lately. Possible dependency update issue?
